py/desisim/scripts/quickgen.py

```diff
@@ -1,109 +1,127 @@
+    parser.add_argument('--nstart', type=int, default=0,help='starting spectra # for simulation 0-4999')
+    parser.add_argument('--spectrograph',type=int, default=None,help='Spectrograph no. 0-9')
+    parser.add_argument('--config', type=str, default='desi', help='specsim configuration')
+    parser.add_argument('-s','--seed', type=int, default=0,  help="random seed")
+    # Only produce uncalibrated output
+    parser.add_argument('--frameonly', action="store_true", help="only output frame files")
+
+    # Moon options if bright or gray time
     parser.add_argument('--moon-phase', type=float,  help='moon phase (0=full, 1=new)', default=None, metavar='')
     parser.add_argument('--moon-angle', type=float,  help='separation angle to the moon (0-180 deg)', default=None, metavar='')
     parser.add_argument('--moon-zenith', type=float,  help='zenith angle of the moon (0-90 deg)', default=None, metavar='')
 
+    # Brick options
+    parser.add_argument('-b','--brickname', type=str, help='unique output brickname suffix', metavar='')
+    parser.add_argument('--objtype', type=str,  help='ELG, LRG, QSO, BGS, MWS, WD, DARK_MIX, or BRIGHT_MIX', default='DARK_MIX', metavar='')
+    parser.add_argument('-a','--airmass', type=float,  help='airmass', default=1.25, metavar='') # Science Req. Doc L3.3.2
+    parser.add_argument('-e','--exptime', type=float,  help='exposure time (s)', default=None,metavar='')
+    parser.add_argument('-o','--outdir', type=str,  help='output directory', default='.', metavar='')
+    parser.add_argument('-v','--verbose', action='store_true', help='toggle on verbose output')
+    parser.add_argument('--outdir-truth', type=str,  help='optional alternative output directory for truth files', metavar='')
+
+    # Object type specific options
+    parser.add_argument('--zrange-qso', type=float, default=(0.5, 4.0), nargs=2, metavar='',
+                        help='minimum and maximum redshift range for QSO')
+    parser.add_argument('--zrange-elg', type=float, default=(0.6, 1.6), nargs=2, metavar='',
+                        help='minimum and maximum redshift range for ELG')
+    parser.add_argument('--zrange-lrg', type=float, default=(0.5, 1.1), nargs=2, metavar='',
+                        help='minimum and maximum redshift range for LRG')
+    parser.add_argument('--zrange-bgs', type=float, default=(0.01, 0.4), nargs=2, metavar='',
+                        help='minimum and maximum redshift range for BGS')
+    parser.add_argument('--rmagrange-bgs', type=float, default=(15.0, 19.5), nargs=2, metavar='',
+                        help='Minimum and maximum BGS r-band (AB) magnitude range')
+    parser.add_argument('--sne-rfluxratiorange', type=float, default=(0.1, 1.0), nargs=2, metavar='',
+                        help='r-band flux ratio of the SNeIa spectrum relative to the galaxy')
+    parser.add_argument('--add-SNeIa', action='store_true', help='include SNeIa spectra')
+
```

Comment:
Please add some comment documentation about:
* how to run this in "quickgen mode", outputting frame files, cframe files, etc.
* how to run this in "quickbrick mode", outputting only bricks
* which options apply to only one or the other vs. both